tox.ini

```diff
@@ -64,7 +64,7 @@ deps=
 changedir=pkgpanda/build/tests
 commands=
   bash -c "cd ../../../ && ./prep_local"
-  py.test -vv {env:CI_FLAGS:} --basetemp={envtmpdir} {posargs}
+  py.test -rs -vv {env:CI_FLAGS:} --basetemp={envtmpdir} {posargs}
```

Comment:
Unfortunately, this is the only form according to py.test --help
On Sat, Aug 6, 2016 at 12:28 Cody Maloney <notifications@github.com> wrote:

> In tox.ini <https://github.com/dcos/dcos/pull/496#discussion_r73790834>:
>
> > @@ -64,7 +64,7 @@ deps=
> >  changedir=pkgpanda/build/tests
> >  commands=
> >    bash -c "cd ../../../ && ./prep_local"
> > -  py.test -vv {env:CI_FLAGS:} --basetemp={envtmpdir} {posargs}
> > +  py.test -rs -vv {env:CI_FLAGS:} --basetemp={envtmpdir} {posargs}
>
> do these have more descriptive long forms?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dcos/dcos/pull/496/files/ac0a26988424a5bf370404e62fd7b3e0f015757c#r73790834>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AACDwxl6T4OUmlw3q1rkfZqnSI9kU2Fcks5qdOA9gaJpZM4JeFg7>
> .
>